easy/5.js

```diff
@@ -1,5 +1,7 @@
 function removeNull(arr) {
-    
+    return arr.filter(function(el){

```

Comment:
If you are using ES6 method of array, why you skipped array function in this example?